bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/protocol/TelenotCommand.java

```diff
@@ -0,0 +1,283 @@
+        strDayOfWeek = (strDayOfWeek.length() == 1 ? "0" + strDayOfWeek : strDayOfWeek);
+        String strHour = Integer.toHexString(hour);
+        strHour = (strHour.length() == 1 ? "0" + strHour : strHour);
+        String strMinute = Integer.toHexString(minute);
+        strMinute = (strMinute.length() == 1 ? "0" + strMinute : strMinute);
+        String strSecond = Integer.toHexString(second);
+        strSecond = (strSecond.length() == 1 ? "0" + strSecond : strSecond);
+
+        String msg = "680B0B6873010750" + strYear + strDayOfWeek + strMonth + strDay + strHour + strMinute + strSecond;
+        msg = msg + checksum(msg) + "16";
+
+        return new TelenotCommand(msg, "set date/time msg: " + msg);
+    }
+
+    /**
+     * Construct an Telenot command to acknowledge that a received CRC message was valid.
+     *
+     * @return TelenotCommand object containing the constructed command
+     */
+    public static TelenotCommand confirmACK() {
+        return new TelenotCommand(COMMAND_CONF_ACK, "Confirm_ACK");
+    }
+
+    /**
+     * Construct an Telenot command to acknowledge that a received CRC message was valid.
+     *
+     * @return TelenotCommand object containing the constructed command
+     */
+    public static TelenotCommand sendNorm() {
+        return new TelenotCommand(COMMAND_SEND_NORM, "SEND_NORM");
+    }
+
+    /**
+     * Construct an Telenot command to return used cntacts.
+     *
+     * @return TelenotCommand object containing the constructed command
+     */
+    public static TelenotCommand sendUsedState() {
+        return new TelenotCommand(COMMAND_USED_STATE, "Used Contacts");
+    }
+
+    /**
+     * Construct an Telenot command to disarm area.
+     *
+     * @param address The SB area number (1-8) for the command.
+     * @return TelenotCommand object containing the constructed command
+     * @throws IllegalArgumentException
+     */
+    public static TelenotCommand disarmArea(int address) throws IllegalArgumentException {
+        if (address < 1 || address > 8) {
+            throw new IllegalArgumentException("Invalid parameter(s)");
+        }
+        String hex = Integer.toHexString(1320 + (address * 8));
+        String msg = "6809096873010502000" + hex + "02E1";
+        msg = msg + checksum(msg) + "16";
+        return new TelenotCommand(msg, "DISARM security area msg: " + msg);
+    }
+
+    /**
+     * Construct an Telenot command to internal arm area.
+     *
+     * @param address The SB area number (1-8) for the command.
+     * @return TelenotCommand object containing the constructed command
+     * @throws IllegalArgumentException
+     */
+    public static TelenotCommand intArmArea(int address) throws IllegalArgumentException {
+        if (address < 1 || address > 8) {
+            throw new IllegalArgumentException("Invalid parameter(s)");
+        }
+        String hex = Integer.toHexString(1321 + (address * 8));
+        String msg = "6809096873010502000" + hex + "0262";
```

Comment:
`6809096873010502000` seem sto be a constant prefix. Introduce a `private static final String` field for that.